scaffolding-chef-infra/lib/linux/scaffolding.sh

```diff
@@ -6,15 +6,15 @@ fi
 scaffolding_load() {
   : "${scaffold_chef_client:=chef/chef-client}"
   : "${scaffold_chef_dk:=chef/chef-dk}"
-  : "${scaffold_cacerts:=}"
+  : "${scaffold_cacerts:=core/cacerts}"
   : "${scaffold_policyfile_path:=$PLAN_CONTEXT/../policyfiles}"
   : "${scaffold_data_bags_path:=$PLAN_CONTEXT/../data_bags}"
 
   pkg_deps=(
     "${pkg_deps[@]}"
     "${scaffold_chef_client}"
   )
-  if [ -n "${scaffold_cacerts}" ]; then
+  if [ "${scaffold_cacerts}" != "core/cacerts" ]; then
```

Comment:
This logic is backwards, we should short-circuit as so:

```
  if [ "${scaffold_cacerts}" == "core/cacerts" ]; then
    pkg_deps+=("core/cacerts")
  else
    pkg_deps+=("${scaffold_cacerts}")
```

Short circuiting this way is a tiny bit faster for the happy path (the default), and also easier to reason about the logic (less error prone).